src/forms/renderers/OAuth.tsx

```diff
@@ -168,8 +218,6 @@ const OAuthproviderRenderer = ({
     });
 
     const openPopUp = async () => {
-        // As we start clear out errors and current credentials
-        removeCofig();
```

Comment:
I initially added this as a belts with suspenders kind of approach. However, doing this causes some weird edge cases. Also, it means that if a users _starts_ to reauthenticate but then decide they do not want to... they would be stuck.